rosbag2_compression/test/rosbag2_compression/test_sequential_compression_reader.cpp

```diff
@@ -76,11 +76,11 @@ class SequentialCompressionReaderTest : public Test
     rosbag2_storage::BagMetadata metadata;
     metadata.version = 4;
     metadata.relative_file_paths = {
-      "bagfile_0.zstd",
-      "bagfile_1.zstd"
+      "bagfile_0.fake_comp",
+      "bagfile_1.fake_comp"
     };
     metadata.topics_with_message_count.push_back({{topic_with_type_}, 1});
-    metadata.compression_format = "zstd";
+    metadata.compression_format = "fake_comp";
```

Comment:
`fake_comp` is a constant in the compression_writer test. Does it make sense to extract it from both?